packages/app-desktop/gui/NoteList/NoteList2.tsx

```diff
@@ -302,6 +302,7 @@ const NoteList = (props: Props) => {
 			onKeyUp={onKeyUp}
 			onDrop={onDrop}
 			onContextMenu={onContainerContextMenu}
+			id='search-results'
```

Comment:
```suggestion
			id='notes-list'
```
Since this `<div>` is also used for displaying the notes list (when not searching), an ID of `notes-list` might make more sense.